src/Configuration/src/ConfigServerBase/ConfigServerConfigurationProvider.cs

```diff
@@ -246,10 +246,12 @@ internal ConfigEnvironment DoLoad(bool updateDictionary = true)
                     else
                     {
                         // Single, server make Config Server URI from settings
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
@dtillman could you please take a look at this section? It wasn't immediately obvious to me how to update this section to use non-obsolete methods